src/test/kotlin/uk/gov/justice/digital/hmpps/managerecallsapi/documents/dossier/DossierContextFactoryTest.kt

```diff
@@ -98,6 +100,96 @@ class DossierContextFactoryTest {
 
     val result = underTest.createContext(recallId)
 
-    assertThat(result, equalTo(DossierContext(recall, currentPrisonName, currentPrisonIsWelsh, 1, STANDARD)))
+    assertThat(result, equalTo(DossierContext(recall, currentPrisonName, currentPrisonIsWelsh, 1)))
+  }
+
+  @Test
+  fun `get TableOfContentsContext for standard recall doesnt calculate recall length`() {
+    val recallId = ::RecallId.random()
+    val nomsNumber = NomsNumber("nomsNumber")
+    val currentPrison = PrisonId("AAA")
+    val currentPrisonName = PrisonName("Current Prison Name")
+    val currentPrisonIsWelsh = Random.nextBoolean()
+
+    val recall = Recall(
+      recallId,
+      nomsNumber,
+      ::UserId.random(),
+      OffsetDateTime.now(),
+      FirstName("Barrie"),
+      null,
+      LastName("Badger"),
+      CroNumber("ABC/1234A"),
+      LocalDate.of(1999, 12, 1),
+      currentPrison = currentPrison,
+      recommendedRecallType = STANDARD,
+      bookingNumber = "A1234"
+    )
+
+    every { recallRepository.getByRecallId(recallId) } returns recall
+    every { prisonLookupService.getPrisonName(currentPrison) } returns currentPrisonName
+    every { prisonLookupService.isWelsh(currentPrison) } returns currentPrisonIsWelsh
+    every { documentRepository.findLatestVersionedDocumentByRecallIdAndCategory(recallId, DocumentCategory.DOSSIER) } returns null
+
+    val result = underTest.createContext(recallId).getTableOfContentsContext()
+
+    assertThat(
+      result,
+      equalTo(
+        TableOfContentsContext(
+          recall.prisonerNameOnLicense(),
+          RecallDescription(STANDARD, null),
+          currentPrisonName,
+          "A1234",
+          1
+        )
+      )
+    )
+  }
+
+  @Test
+  fun `get TableOfContentsContext for Fixed recall calculates recall length`() {
```

Comment:
Personally I'd prefer us to keep calling these `Fixed Term Recall` rather than start to abbreviate.  
(Separate of course but in the tickets we also have random re-ordering of words creeping in - just making the picture unnecessarily confusing, IMO).

Reply:
this one I can do 🤣